### PR TITLE
Temporary counter (testApp)

### PR DESCRIPTION
**Pull request by pamelalozano16**
##PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the new behavior?

Hi guys I created this counter with a listener on the client, so whenever I update the counter document in the database it changes, everyone can see the number but only if you are signed in you can vote.
To run it go to the testApp/web-start folder and run in the terminal `firebase serve --only hosting`
## Other information
Screenshot 2020-07-21 at 15 02 32 - Display 2

### CODE REVIEW

**Comment by brettapeters** [comment]
Can you move this into the App Engine app instead of a separate app served from Firebase hosting? I think you just need to move index.html, index.js, and public/ to src/main/webapp/

File: src/main/webapp/index.html [hunk 1]
```diff
@@ -1,18 +1,118 @@
...
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License
+-->
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="description" content="Learn how to use the Firebase platform on the Web">
```
Comment (seanjohnite) [inline src/main/webapp/index.html:18]: remove?

File: src/main/webapp/index.html [hunk 2]
```diff
@@ -1,18 +1,118 @@
...
+      <div id="must-signin-snackbar" class="mdl-js-snackbar mdl-snackbar">
+        <div class="mdl-snackbar__text"></div>
+        <button class="mdl-snackbar__action" type="button"></button>
+      </div>
+
+    </div>
+  </main>
+</div>
+
+ <!-- Insert these scripts at the bottom of the HTML, but before you use any Firebase services -->
```
Comment (seanjohnite) [inline src/main/webapp/index.html:104]: remove?

File: src/main/webapp/index.html (hunk 1 above)
Comment (pamelalozano16) [reply src/main/webapp/index.html]: Done! :)

File: src/main/webapp/index.html (hunk 2 above)
Comment (pamelalozano16) [reply src/main/webapp/index.html]: Thanks! done

**Pull request merged**